ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/Toml.kt

```diff
@@ -60,7 +60,9 @@ public open class Toml(
     }
 
     override fun <T> encodeToString(serializer: SerializationStrategy<T>, value: T): String {
-        TODO("Not yet implemented")
+        val toml = TomlMainEncoder.encode(serializer, value, inputConfig, outputConfig, serializersModule)
```

Comment:
why do we need inputConfig right here? I thought we will split it

Reply:
We did, but nodes still require it in the constructor

Reply:
Ah, I got it. We create tree first even for encoding, sure